### DISCUSSION TITLE
VNC viewer and Termux

### DISCUSSION THREAD

**Discussion by ghxstweb** [termux/termux-app#3405] [General]
Hellow all, I was experimenting with Termux to get a Linux UI up and running through Termux's proot-distro program. I went through all of the steps exactly shown in this video : https://www.youtube.com/watch?v=nbdIjoN2gWA (The only difference is that your vncstart file may referene /usr/bin/startplasma-x11 instead of /bin/startplasma-x11)

However, when I attempt to run my vnc server, my phone turns off. What is going on here? Is this a security feature or something? Seems like its a power management thing, right? It just completely shuts off, and then turns itself back on (restarts). I would really appreciate any advice or help on this issue. Thanks

**Comment by twaik** [comment]
It looks like phone has kernel panic or something simillar. Termux itself has no permissions to reboot your device.

**Comment by agnostic-apollo** [comment]
Run `adb logcat > logcat.txt` from pc and then trigger crash and then check logs, it may show the reason/exception at end. Termux normally can't fix device specific crash issues.
